Trading/Mode/grid_trading_mode/grid_trading.py

```diff
@@ -329,15 +344,18 @@ def read_config(self):
             self.health_check_interval_secs = self.symbol_trading_config.get(
                 self.trading_mode.CONFIG_FUNDS_REDISPATCH_INTERVAL, self.funds_redispatch_interval
             ) * commons_constants.HOURS_TO_SECONDS
-        self.compensate_for_missed_mirror_order = self.symbol_trading_config.get(
-            self.trading_mode.COMPENSATE_FOR_MISSED_MIRROR_ORDER, self.compensate_for_missed_mirror_order
-        )
         self.enable_trailing_up = self.symbol_trading_config.get(
             self.trading_mode.CONFIG_ENABLE_TRAILING_UP, self.enable_trailing_up
         )
         self.enable_trailing_down = self.symbol_trading_config.get(
             self.trading_mode.CONFIG_ENABLE_TRAILING_DOWN, self.enable_trailing_down
         )
+        self.use_order_by_order_trailing = self.symbol_trading_config.get(
+            self.trading_mode.CONFIG_ORDER_BY_ORDER_TRAILING, self.use_order_by_order_trailing
+        )
+        self.compensate_for_missed_mirror_order = self.symbol_trading_config.get(
+            self.trading_mode.COMPENSATE_FOR_MISSED_MIRROR_ORDER, self.use_order_by_order_trailing # use use_order_by_order_trailing as default value as compensate_for_missed_mirror_order is required for order by order trailing 
```

Comment:
What if we want to enable compensate_for_missed_mirror_order without use_order_by_order_trailing?